doc/howto/cluster_manage.md

```diff
@@ -5,79 +5,115 @@ discourse: 11330
 
 ## Evacuating and restoring cluster members
 
-Whether it's for routine maintenance like applying system updates requiring
-a reboot or to perform hardware changes, you may sometimes want to empty a
-given server of all its instances.
+There are scenarios where you might need to empty a given cluster member of all its instances (for example, for routine maintenance like applying system updates that require a reboot, or to perform hardware changes).
+
+To do so, use the `lxc cluster evacuate` command.
+This command migrates all instances on the given server, moving them to other cluster members.
+The evacuated cluster member is then transitioned to an "evacuated" state, which prevents the creation of any instances on it.
 
-This can be done using `lxc cluster evacuate <NAME>` which will migrate all
-instances on that server, moving them to other cluster members. The evacuated
-cluster member will be transitioned to an "evacuated" state which will prevent
-the creation of any instances on it.
+You can control how each instance is moved through the [`cluster.evacuate`](instance-configuration) instance configuration key.
+Instances are shut down cleanly, respecting the `boot.host_shutdown_timeout` configuration key.
+
+When the evacuated server is available again, use the `lxc cluster restore` command to move the server back into a normal running state.
+This command also moves the evacuated instances back from the servers that were temporarily holding them.
+
+(cluster-manage-delete-members)=
+## Deleting cluster members
 
-Once maintenance is complete, `lxc cluster restore <NAME>` will move the server
-back into a normal running state and will move its instances back from the servers
-that were temporarily holding them.
+To cleanly delete a member from the cluster, use the `lxc cluster remove` command.
 
-The behavior for a given instance can be configured through the `cluster.evacuate`
-instance configuration key. Instances will be shutdown cleanly, respecting the
-`boot.host_shutdown_timeout` configuration key.
+You can only delete members that are online and that don't have any instances located on them.
+If required, you can force-remove a cluster member (for example, if the server is permanently offline or had to be reinstalled).
+To do so, enter the following command:
+
+    lxc cluster remove --force <member_name>
+
+```{warning}
+Force-removing a cluster member might leave the member in an inconsistent state.
+For example, the storage pool on the member will not be removed, which might cause problems when re-initializing LXD later.
+```
+
+## Upgrading cluster members
+
+To upgrade a cluster, you must upgrade all of its members.
+All members must be upgraded to the same version of LXD.
+
+To upgrade a single member, simply upgrade the `lxd`/`lxc` binaries on the host and restart the LXD daemon.
```

Comment:
By mentioning the specific binaries here it suggests that this is somehow different from upgrading LXD in general.
By only updating the binaries rather than the LXD package the resulting cluster may not be operational as it may be missing dependencies that the package would have installed.

How about `To upgrade a single member, simply upgrade the LXD package on the host and restart the LXD daemon.`